### DISCUSSION TITLE
Speed test!

### DISCUSSION THREAD

**Discussion by fakemanhk** [fakemanhk/openwrt-jp-ipoe#4] [Show and tell]
Speed test is the most direct way to tell the actual difference between IPoe & PPPoE, which explains why we (including ISP) want to use IPoE rather than PPPoe.

Testing environment:

- IPoE Router - Linksys WRT3200ACM with OpenWrt 22.03.2
- PPPoE Router - pfSense VM hosting at Synology DS1621+ w/AMD V1500B 2-core assigned (PPPoE is single threaded so it doesn't matter)
- Testing terminal - HP Dragonfly Chromebook (i7-1265U + 32GB RAM), with USB Realtek 2.5G ethernet

For both tests I disabled WiFi, only using ethernet direct connection between testing terminal & routers.

The first one is [Ookala speed test](https://www.speedtest.net/):
![Screenshot 2023-02-04 05 03 14](https://user-images.githubusercontent.com/21307353/216702953-658f4222-b860-422b-9a02-38dd48f47706.png)

Second one is[ みなのネッド回線速度](https://minsoku.net/):
![Screenshot 2023-02-04 04 56 27](https://user-images.githubusercontent.com/21307353/216702435-5ab4eed2-e963-484e-a1d9-bd819239d03a.png)

As observed, using IPoE can yeild to a much faster speed when compared with PPPoE. Note that my location is in Kanagawa which is not in central Tokyo, the PPPoE performance isn't too bad already. My tests were carried out at 5AM local time to ensure less people fighting for network resources.

As mentioned above PPPoE has a single threaded nature so from my past experience I have never seen it producing > 750Mb/s, not to mention the Linksys WRT3200ACM (it can only be slower), but now with IPoE WRT3200ACM is able to give me close to 900Mb/s download speed which is something I can never imagine.

**Comment by TimOfAllTrades** [comment]
Followed your advice and got the following down/up speeds.  OCN 1g docomo hikari

Pppoe congested time 7pm
200-300 / 200-300
Ipv4 over ipv6 congested time
300-400 / 300-400

Pppoe uncongested and ip4 over ipv6 uncontested time are about the same at
500-600 / 500-600

Ipv6 is about
600-700 / 600 - 700

The most notable difference is Ipv4 during congested times.

**Reply by fakemanhk** [reply]
That's nice! Not just bandwidth, the ping/jitter also got improvements (noticeable during online gaming session)
